tests/konsist/build.gradle.kts

```diff
@@ -32,3 +32,8 @@ dependencies {
     testImplementation(projects.libraries.architecture)
     testImplementation(projects.libraries.designsystem)
 }
+
+// Make sure Konsist tests are always run
+tasks.withType<Test>().configureEach {
+    outputs.upToDateWhen { false }
+}
```

Comment:
Konsist test are not always run because the `konsist` module does not depend on any other modules, except `projects.libraries.architecture` and `projects.libraries.designsystem` (see the lines above) because we need access to some external types.
Maybe clarify why this patch is needed?

Reply:
Done in https://github.com/vector-im/element-x-android/pull/1590/commits/fb0bf385fc2278cf809b6415f225345324bbbb5f.